pom.xml

```diff
@@ -104,6 +104,12 @@
 			<groupId>com.h2database</groupId>
 			<artifactId>h2</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.postgresql</groupId>
+			<artifactId>postgresql</artifactId>
+			<version>42.5.1</version>
```

Comment:
You can get over without setting up the version, as spring controls driver versions.